mypy/test/helpers.py

```diff
@@ -312,6 +255,55 @@ def retry_on_error(func: Callable[[], Any], max_wait: float = 1.0) -> None:
             time.sleep(wait_time)
 
 
+class AssertionFailure(Exception):
+    """Exception used to signal failed test cases."""
+    def __init__(self, s: Optional[str] = None) -> None:
+        if s:
+            super().__init__(s)
+        else:
+            super().__init__()
+
+
+def assert_true(b: bool, msg: Optional[str] = None) -> None:
```

Comment:
Add comment about `assert_true` and `assert_false` being redundant (we could just use the assert statement).